### ISSUE TITLE
How could I fine tune the pre-trained model using my data ?

### ISSUE THREAD

**Issue by xuewenyuan** [jugg1024/Text-Detection-with-FRCN#3]

**Comment by jugg1024** [comment]
Hi, i wrote it on readme.md， you should prepare your dataset(both data and label) in the format of pascal_voc annotation, examples can be seen in: ./datasets/script/format_annotation.py format_coco_text()
then training script here：./experiments/scripts/faster_rcnn_end2end.sh
